packages/cubejs-base-driver/src/driver.interface.ts

```diff
@@ -152,12 +157,13 @@ export interface DriverInterface {
   stream?: (table: string, values: unknown[], options: StreamOptions) => Promise<StreamTableData>;
   // Some drivers can implement UNLOAD data to external storage
   unload?: (table: string, options: UnloadOptions) => Promise<DownloadTableCSVData>;
+  unloadWithSql?: (table: string, query: string, params: unknown[], options: UnloadOptions) => Promise<DownloadTableCSVData>;
```

Comment:
📓 We added only unloadWithSql because snowflake uses absolutely different flow compared to databricks

https://docs.snowflake.com/en/sql-reference/sql/desc-result.html#:~:text=Describes%20the%20columns%20in%20the,limited%20period%20has%20not%20elapsed.